stdlib/public/Concurrency/SourceCompatibilityShims.swift

```diff
@@ -298,279 +298,9 @@ extension Task where Failure == Never {
-  public mutating func add(
-    operation: __owned @Sendable @escaping () async throws -> ChildTaskResult
-  ) async -> Bool {
-    return self.addTaskUnlessCancelled {
-      try await operation()
-    }
-  }
-
-  @available(*, unavailable, message: "Unavailable in task-to-thread concurrency model", renamed: "addTask(operation:)")
-  public mutating func spawn(
-    priority: TaskPriority? = nil,
-    operation: __owned @Sendable @escaping () async throws -> ChildTaskResult
-  ) {
-    fatalError("Unavailable in task-to-thread concurrency model")
-  }
-
-  @available(*, deprecated, renamed: "addTask(operation:)")
-  @_alwaysEmitIntoClient
-  public mutating func spawn(
-    operation: __owned @Sendable @escaping () async throws -> ChildTaskResult
-  ) {
-    addTask(operation: operation)
-  }
-
-  @available(*, unavailable, message: "Unavailable in task-to-thread concurrency model", renamed: "addTaskUnlessCancelled(operation:)")
-  public mutating func spawnUnlessCancelled(
-    priority: TaskPriority? = nil,
-    operation: __owned @Sendable @escaping () async throws -> ChildTaskResult
-  ) -> Bool {
-    fatalError("Unavailable in task-to-thread concurrency model")
-  }
-
-  @available(*, deprecated, renamed: "addTaskUnlessCancelled(operation:)")
-  @_alwaysEmitIntoClient
-  public mutating func spawnUnlessCancelled(
-    operation: __owned @Sendable @escaping () async throws -> ChildTaskResult
-  ) -> Bool {
-    addTaskUnlessCancelled(operation: operation)
-  }
-
-  @available(*, unavailable, message: "Unavailable in task-to-thread concurrency model", renamed: "addTask(operation:)")
-  public mutating func async(
-    priority: TaskPriority? = nil,
-    operation: __owned @Sendable @escaping () async throws -> ChildTaskResult
-  ) {
-    fatalError("Unavailable in task-to-thread concurrency model")
-  }
-
-  @available(*, deprecated, renamed: "addTask(operation:)")
-  @_alwaysEmitIntoClient
-  public mutating func async(
-    operation: __owned @Sendable @escaping () async throws -> ChildTaskResult
-  ) {
-    addTask(operation: operation)
-  }
-
-  @available(*, unavailable, message: "Unavailable in task-to-thread concurrency model", renamed: "addTaskUnlessCancelled(operation:)")
-  public mutating func asyncUnlessCancelled(
-    priority: TaskPriority? = nil,
-    operation: __owned @Sendable @escaping () async throws -> ChildTaskResult
-  ) -> Bool {
-    fatalError("Unavailable in task-to-thread concurrency model")
-  }
-
-  @available(*, deprecated, renamed: "addTaskUnlessCancelled(operation:)")
-  @_alwaysEmitIntoClient
-  public mutating func asyncUnlessCancelled(
-    operation: __owned @Sendable @escaping () async throws -> ChildTaskResult
-  ) -> Bool {
-    addTaskUnlessCancelled(operation: operation)
-  }
-}
-#endif
+// NOTE: We had to move sources touching TaskGroup addTask APIs into TaskGroup+addTask.swift.gyb
+// due to the build having trouble with using the generated source in the same module
+// rdar://145171772
```

Comment:
Build problem #1, I can't figure out how to get the gyb generated file included in the module 🤔